From caf8c2a22b860effdfa7511f0a035d8b668e8a85 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ga=C3=ABl=20Bonithon?= <gael@xfce.org>
Date: Fri, 1 Apr 2022 23:56:33 +0200
Subject: [PATCH 1/2] Bump minimal required libxfce4util to 4.17.2

`xfce_str_replace()` added in libxfce4util!24 requires libxfce4util
4.17.2.
---
 configure.ac.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac.in b/configure.ac.in
index e9a4d75..3fef46a 100644
--- a/configure.ac.in
+++ b/configure.ac.in
@@ -126,7 +126,7 @@ dnl *** Check for required packages ***
 dnl ***********************************
 XDT_CHECK_PACKAGE([GLIB], [glib-2.0], [2.50.0])
 XDT_CHECK_PACKAGE([GTK3], [gtk+-3.0], [3.22.0])
-XDT_CHECK_PACKAGE([LIBXFCE4UTIL], [libxfce4util-1.0], [4.15.6])
+XDT_CHECK_PACKAGE([LIBXFCE4UTIL], [libxfce4util-1.0], [4.17.2])
 
 dnl ****************************************************************
 dnl *** Optional support for system information based on glibtop ***
-- 
2.35.1

